src/themes/colors.ts

```diff
@@ -16,6 +16,7 @@ export const colors = {
   warningLight: palette.yellowLight, //text
   shadow: palette.shadow,
   modalBackdrop: palette.grayTransparent,
+  border: palette.greyLighter,
```

Comment:
Will this be the border color for the majority of components in our design? If we proceed with this name, can we use borderLight, borderDark appropriately on other components?